concourse/tasks/delete_admin.yml

```diff
@@ -3,32 +3,46 @@ platform: linux
 image_resource:
   type: docker-image
   source:
-    repository: governmentpaas/cf-uaac
+    repository: governmentpaas/cf-cli
+    tag: feature_126965751_move_cf_cli_to_alpine
 inputs:
   - name: paas-cf
   - name: cf-secrets
   - name: cf-manifest
   - name: admin-creds
   - name: bosh-CA
+  - name: config
 run:
   path: sh
   args:
     - -e
+    - -u
     - -c
     - |
-      NAME=$(cat admin-creds/username)
-      if [ "${ENABLE_ADMIN_USER_CREATION}" == "false" ]; then
+      if [ "${ENABLE_ADMIN_USER_CREATION:-}" == "false" ]; then
         echo "Temporary user creation is disabled (ENABLE_ADMIN_USER_CREATION=${ENABLE_ADMIN_USER_CREATION}). Skipping."
         exit 0
       fi
       ./paas-cf/concourse/scripts/import_bosh_ca.sh
+      . ./config/config.sh
 
-      VAL_FROM_YAML=$(pwd)/paas-cf/concourse/scripts/val_from_yaml.rb
-      UAA_ADMIN_CLIENT_PASS=$($VAL_FROM_YAML secrets.uaa_admin_client_secret cf-secrets/cf-secrets.yml)
-      UAA_ENDPOINT=$($VAL_FROM_YAML properties.uaa.url cf-manifest/cf-manifest.yml)
+      NAME=$(cat admin-creds/username)
 
       echo "Removing user ${NAME}"
-      uaac target "${UAA_ENDPOINT}"
-      uaac token client get admin -s "${UAA_ADMIN_CLIENT_PASS}"
 
-      uaac user delete "${NAME}"
+      echo | cf login -a ${API_ENDPOINT} -u ${CF_ADMIN} -p ${CF_PASS}
+
+      cf delete-user "${NAME}" -f
+
+      echo "Removing leaked ${PREFIX} users"
```

Comment:
@richardc @keymon do you have any thoughts about whether we'd remove the following scripting once the previously leaked users have been removed from existing persistent environments?

If we do, can we do anything to make it easier for us to remove this in the future, such as adding a `FIXME: remove later` comment and/or breaking up the commits to allow us to revert this part (but retain the `cf delete-user` above)?